_episodes/03-create.md

```diff
@@ -618,6 +485,104 @@ but it does find the copy in `thesis` that we didn't delete.
 > {: .solution}
 {: .challenge}
 
+## Removing files and directories
+
+Returning to the `data-shell` directory,
+let's tidy up this directory by removing the `quotes.txt` file we created.
+The Unix command we'll use for this is `rm` (short for 'remove'):
+
+~~~
+$ rm quotes.txt
+~~~
+{: .language-bash}
+
+We can confirm the file has gone using `ls`:
+
+~~~
+$ ls quotes.txt
+~~~
+{: .language-bash}
+
+```
+ls: cannot access 'quotes.txt': No such file or directory
+```
+{: .output}
+
+> ## Deleting Is Forever
+>
+> The Unix shell doesn't have a trash bin that we can recover deleted
+> files from (though most graphical interfaces to Unix do).  Instead,
+> when we delete files, they are unlinked from the file system so that
+> their storage space on disk can be recycled. Tools for finding and
+> recovering deleted files do exist, but there's no guarantee they'll
+> work in any particular situation, since the computer may recycle the
+> file's disk space right away.
+{: .callout}
+
+
+If we try to remove the `thesis` directory using `rm thesis`,
+we get an error message:
+
+~~~
+$ rm thesis
+~~~
+{: .language-bash}
+
+~~~
+rm: cannot remove `thesis': Is a directory
+~~~
+{: .error}
+
+This happens because `rm` by default only works on files, not directories.
+
+`rm` can remove a directory *and all its contents* if we use the 
+recursive flag `-r`, however ...
+
+
+> ## With Great Power Comes Great Responsibility
+>
+> Removing the files in a directory recursively can be a very dangerous
+> operation. If we're concerned about what we might be deleting we can
+> add the "interactive" flag `-i` to `rm` which will ask us for confirmation
+> before each step
+>
+> ~~~
+> $ rm -r -i thesis
+> rm: descend into directory 'thesis'? y
+> rm: remove regular empty file 'thesis/quotations.txt'? y
+> rm: remove directory 'thesis'? y
+> ~~~
+> {: .language-bash}
+>
+> This removes any files in the directory, then the directory itself, asking
+> at each step for you to confirm the deletion.
+{: .callout}
+
+If you haven't already deleted the `thesis` directory using the interactive flag,
+you can delete the directory without any confirmation prompts using the following command:
+
+~~~
+$ rm -r thesis
+~~~
+{: .language-bash}
+
+
+> ## Using `rm` Safely
```

Comment:
I'd move the "Using `rm` Safely" exercise right after the "Deleting Is Forever" callout since at that point we are still dealing with removing files and it would make the "With Great Power Comes Great Responsibility" easier to understand.

Reply:
Yes, good idea.

Reply:
Thanks!